examples/game_of_life/Cargo.toml

```
@@ -11,5 +11,5 @@ log = "0.4.8"
 wasm-logger = "0.2.0"
 yew = { path = "../../yew" }
 rand = { version = "0.7.3", features = ["getrandom"] }
-getrandom = { version = "0.1.14", features = ["wasm-bindgen"] }
+getrandom = { version = "0.2.0", features = ["wasm-bindgen"] }
```

```suggestion
getrandom = { version = "0.2.0", features = ["js"] }
```
Looking at their source it looks like they renamed the feature ([before](https://github.com/rust-random/getrandom/blob/v0.1.15/src/lib.rs#L244) - [after](https://github.com/rust-random/getrandom/blob/v0.2.0/src/lib.rs#L216))

---

Yeah, you're totally right. Unfortunately `rand` still depends on `getrandom ^0.1.1` and I think we'll need to wait until `rand` is published with support for `getrandom ^0.2`

---

Yup. I was just browsing around and noticed this would probably break the example once the PR lands.